views/view-teams.ejs

```diff
@@ -0,0 +1,15 @@
+<% for(var i = 0; i < Teams.length; i++) { %>
```

Comment:
Add a comment or add a change to the name of the files to distinguish between the instructor and student view team files